packages/components/tree-v2/src/tree-node.vue

```diff
@@ -79,27 +82,30 @@ const emit = defineEmits(treeNodeEmits)
 const tree = inject(ROOT_TREE_INJECTION_KEY)
 const ns = useNamespace('tree')
 
-const indent = computed(() => {
-  return tree?.props.indent ?? 16
-})
+const indent = computed(() => tree?.props.indent ?? 16)
+const icon = computed(() => tree?.props.icon ?? CaretRight)
 
-const icon = computed(() => {
-  return tree?.props.icon ?? CaretRight
-})
+const getNodeClass = (node: TreeNode) => {
+  const nodeClassFunc = tree?.props.props.class
+  if (!nodeClassFunc) return {}
 
-const handleClick = (e: MouseEvent) => {
-  emit('click', props.node, e)
-}
+  let className
+  if (isFunction(nodeClassFunc)) {
+    const { data } = node
+    className = nodeClassFunc(data, node)
+  } else {
+    className = nodeClassFunc
+  }
 
-const handleDrop = (e: DragEvent) => {
-  emit('drop', props.node, e)
+  return isString(className) ? { [className]: true } : className
 }
-const handleExpandIconClick = () => {
-  emit('toggle', props.node)
-}
-const handleCheckChange = (value: CheckboxValueType) => {
+
+const handleClick = (e: MouseEvent) => emit('click', props.node, e)
```

Comment:
For a function like this that does not directly return a value, I don't think it's necessary to refactor the abbreviation. I believe the original structure looks clearer.